clang/docs/checkedc/3C/README.md

```diff
@@ -48,3 +50,66 @@ that 3C will contain the core inference logic, while 5C will add
 features to enhance developer productivity. If you'd like more
 information about 5C, please contact us at
 info@correctcomputation.com.
+
+### Which `checkedc-clang` repository to use?
+
+We typically recommend that serious 3C users use CCI's repository to
+get 3C fixes and enhancements sooner, but in some scenarios, you may
+be better off with Microsoft's repository. Here, in detail, are the
+factors we can think of that might affect your decision (as of
+February 2021):
+
+- CCI strives to merge changes reasonably quickly from Microsoft's
+  repository to CCI's, but most 3C-specific changes are made first in
+  CCI's repository and merged to Microsoft's in batches every few
+  months. Thus, CCI's repository typically gives you a significantly
+  newer version of 3C, while Microsoft's repository typically gives
+  you a somewhat newer version of the rest of the Checked C codebase,
+  including some shared code used by 3C. The implication of that last
+  point is that a fix to the shared code in Microsoft's repository
+  would benefit Microsoft's copy of 3C immediately but would not take
+  effect on CCI's copy until the next merge from Microsoft to CCI, so
+  CCI's copy of 3C is not always newer in _all_ respects.
+
+- While the [3C regression tests](CONTRIBUTING.md#testing) are run on
+  every change to either repository, CCI's copy of 3C undergoes
+  additional testing: a set of "benchmark" tests that run nightly,
+  plus manual use by CCI engineers. If a change to CCI's repository
+  passes the regression tests but breaks the additional tests, we will
+  generally fix the problem quickly. But if the same happens in
+  Microsoft's repository, functionality of Microsoft's copy of 3C that
+  is not covered by the regression tests could remain broken for some
+  time. The problem would likely be detected the next time CCI tries
+  to merge from Microsoft, in which case we would try to get a fix
+  into Microsoft's repository reasonably quickly.
+
+- The 3C regression tests are run regularly on Windows on Microsoft's
+  repository but not on CCI's. A change that breaks the regression
+  tests on Windows generally won't be made to Microsoft's repository,
+  but such a change could be made to CCI's and the problem may not be
+  detected until the next time CCI tries to merge to Microsoft (or a
+  user reports the problem to us). So you're less likely to encounter
+  Windows-specific problems with Microsoft's repository.
+
+- On the other hand, some CCI engineers work on Mac OS X and
+  frequently run the regression tests and other manual tests on CCI's
+  copy of 3C on Mac OS X, while we are unaware of any testing of
+  Microsoft's copy of 3C on Mac OS X. So conceivably, you may be less
```

Comment:
Drop "conceivably, "

Reply:
Done.  To ease review, I'm not rewrapping the lines now; I was going to do that as part of the code style pass anyway.